spec/Section 3 -- Type System.md

```diff
@@ -781,26 +1181,67 @@ should only allow such values as enum input values. Otherwise, for most
 transport serializations that do not, strings may be interpreted as the enum
 input value with the same name.
 
+**Type Validation**
+
+Enum types have the potential to be invalid if incorrectly defined.
+
+1. A Enum type must define one or more unique enum values.
```

Comment:
Is there a reason the validation rules are saying "A Enum type" or is it a typo? Happens here and in the "Enum Extensions" type validation section.